File: vertx-server/src/main/java/org/demo/vertx/models/Order.java
package org.demo.vertx.models;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Order {

    private String id;
    private String product;
    private String client;
    private OrderStatut statut;
}

File: website-client/src/app/models/order.ts
export class Order {
  public id: string;
  public client: string;
  public product: string;
  public statut: string;

  constructor(id_: string, client_: string, product_: string, statut_: string) {
    this.id = id_;
    this.client = client_;
    this.product = product_;
    this.statut = statut_;
  }
}
